src/features/ci-cd/hooks/service-installs.ts

```diff
@@ -224,3 +224,138 @@ hooks.addPureHook('PATCH', 'resin', 'device', {
 		}
 	},
 });
+
+hooks.addPureHook('PATCH', 'resin', 'device', {
+	POSTRUN: async ({ api, request }) => {
+		const affectedIds = request.affectedIds!;
+		if (affectedIds.length === 0) {
+			return;
+		}
+
+		// Create supervisor serice installs when device checks in with it's initial version
+		if (request.values.supervisor_version) {
+			await createSupervisorServiceInstallsFromVersion(
+				api,
+				affectedIds,
+				request.values.supervisor_version,
```

Comment:
I think you can skip this as we should never need to create service installs for the currently running release? I think it should only create service installs for the target state of `should_be_managed_by__release` for supervisor releases in the same way it only creates for the target state of `should_be_running__release` for app releases (and ignores `is_running__release`)